### PR TITLE
Fix in theme documentation and ThemeContext

### PR DESCRIPTION
**Pull request by raquelarrojo**
- Fix in `Custom Themes` documentation.
- Added `dateInput` and `select` to DefaultTheme.json.
- Added `textInput` and `select` to Default.schema.json.
- Added `select` to ThemeContext.
- Change `selectedBaseColor` to `selectedFontColor` in toggle's `selectedDisabledFontColor` token in ThemeContext.

### CODE REVIEW

File: docs/src/pages/overview/Overview.jsx [hunk 1]
```diff
@@ -109,8 +109,8 @@ function CustomThemes() {
         customize, using the next structure:
         <SyntaxHighlighter language="javascript" style={docco}>
           {`componentName: {
-  themeInput: "value", 
-}`}
+            themeInput: "value", 
+          }`}
```
Comment (marcialfps) [inline docs/src/pages/overview/Overview.jsx:113]: Remove tabs.
image

File: docs/src/pages/overview/Overview.jsx [hunk 2]
```diff
@@ -125,125 +125,123 @@ function CustomThemes() {
 
       <SyntaxHighlighter language="javascript" style={docco}>
         {`
-  import { DxcButton, ThemeProvider } from "@dxc-technology/halstack-react";
+          import { DxcButton, ThemeProvider } from "@dxc-technology/halstack-react";
```
Comment (marcialfps) [inline docs/src/pages/overview/Overview.jsx:128]: I think we should remove the first tab, looks very alienated on the right.

**Review by marcialfps — CHANGES_REQUESTED** [review]

File: docs/src/pages/overview/Overview.jsx (hunk 1 above)
Comment (raquelarrojo) [reply docs/src/pages/overview/Overview.jsx]: Done!

File: docs/src/pages/overview/Overview.jsx (hunk 2 above)
Comment (raquelarrojo) [reply docs/src/pages/overview/Overview.jsx]: Done!

**Review by marcialfps — APPROVED** [review]
🔥 🔥 🔥

**Pull request merged**